podcasts/AccountViewController+TableView.swift

```diff
@@ -46,8 +49,24 @@ extension AccountViewController: UITableViewDataSource, UITableViewDelegate {
         case .upgradeView:
             let cell = tableView.dequeueReusableCell(withIdentifier: PlusAccountPromptTableCell.reuseIdentifier, for: indexPath) as! PlusAccountPromptTableCell
             cell.updateParent(self)
+            cell.contentSizeUpdated = { [weak self] size in
+                self?.upgradePromptViewSize = size
+            }
+
+            return cell
+
+        case .upgradeAccount:
+            let cell = tableView.dequeueReusableCell(withIdentifier: AccountViewController.actionCellId, for: indexPath) as! AccountActionCell
+            cell.cellLabel.text = "Upgrade Account"
```

Comment:
Should this be localized?

Reply:
Yup! Fixed here: cd0e9a835ed0465c01dbadb8c26706410c5cf1e5